samples/simulation/ising/simple/Program.qs

```diff
@@ -1,14 +1,14 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I recognize this was pre-existing, but may be good to update the copyright header here.
```suggestion
﻿// Copyright (c) Microsoft Corporation.
```